.github/assets/js/shortcut-key.js

```
@@ -0,0 +1,24 @@
+document.addEventListener('DOMContentLoaded', function() {
+  // Detect if the user is on a Mac using modern API with fallback
+  let isMac = false;
+  
+  if (navigator.userAgentData && navigator.userAgentData.platform) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)